packages/mdc-menu/mdc-menu.scss

```diff
@@ -27,10 +27,6 @@ $mdc-menu-scale-duration: .12s;
 .mdc-menu {
```

Comment:
Can we change `background-color: white` to `background-color: $mdc-theme-background`?

(We'll need to add `@import "@material/theme/variables"` to the top of this file)